tests/data/square_map/properties

```diff
@@ -5,8 +5,8 @@ hats_nrows=12
 hats_max_rows=1000000
 hats_order=1
 moc_sky_fraction=0.25000
-hats_builder=hats-import v0.4.1
-hats_creation_date=2024-11-20T19\:32UTC
-hats_estsize=52
+hats_builder=hats-import v0.4.6.dev1+gf00cd7a
```

Comment:
I think this is the only file where I see a released version changing to a dev version.  Intentional?

On that note, is there a reason why all the `properties` file can't reference a released version?  (And does it even matter?)